src/scss/_color-use-cases.scss

```diff
@@ -52,14 +52,6 @@
     'border': 'lemon',
     'text': 'lemon'
 ));
-@include oColorsSetUseCase('o-teaser/theme-paid-post', (
-    'background': 'white',
-    'text': 'black'
-));
-@include oColorsSetUseCase('o-teaser/theme-promoted-content', (
-    'background': 'white',
-    'text': 'black'
-));
```

Comment:
Does removing these make the pull-request a breaking change? My thinking is this would make it a breaking change because these colors would no longer exist.

What do you think @notlee?

Reply:
They were just moved to a different file -- src/scss/_deprecated.scss